### ISSUE TITLE
`terminal_columns': undefined method `winsize' for nil:NilClass

### ISSUE THREAD

**Issue by antonkulaga** [Blahah/transrate#216]
When I install transrate and run transrate command I get misterious
```
`terminal_columns': undefined method `winsize' for nil:NilClass
```
here is the docker file content, so you can reproduce it:
```
FROM ruby:2.2.8

MAINTAINER Anton Kulaga <antonkulaga@gmail.com>

RUN apt-get update && \
 apt-get upgrade -y && \
 apt-get install -y cmake zip wget gcc git build-essential

RUN gem install transrate
RUN transrate --install-deps all

RUN mkdir /data
WORKDIR /data

CMD ["transrate"]
```

**Comment by antonkulaga** [comment]
@blahah any updates?

**Comment by dpryan79** [comment]
For the bioconda recipe and biocontainer I've needed to disable the chunk of code that checks the console size before printing the banner https://github.com/bioconda/bioconda-recipes/pull/18581/files

This solves the issue, though it'd probably be better to simply not bother with the banner. Interestingly, without patching it runs without printing the transrate banner outside of the docker container but in the CI environment, so it seems like the io-container ruby gem needs something that's just not in minimal docker containers.

**Comment by blahah** [comment]
@dpryan79 do you think it's the case that we should just disable the banner stuff to maximise compatibility? Happy to push small fixes if I can.

**Comment by dpryan79** [comment]
@blahah yeah, that'd be the path of least resistance. Just print a small banner with simple print functions if you really want to.

**Comment by blahah** [comment]
So much for trying to add a little flare to the day of bioinformaticians :joy: 

OK. I'll carve out the banner (everything with colours) and leave it plain. Usability and composability do more social good than fun I suppose.
